README.md

```diff
@@ -104,24 +58,31 @@ Which font weight of Operator Mono do you use? Also note difference between Scre
 > NOTE: Because _Operator Mono_ is not a free font, you must have the original font files. This utility
 > will merge the ligature definitions into a copy of the original font. The new font family is named _Operator Mono Lig_ so you can install it side-by-side with the original font.
 
-## Prerequisites
+# 🛠 How to Install
+
+## Using Unix-based/Windows OS with Python installed:
```

Comment:
This title is pretty long and could probably be shortened.

Reply:
How about "Using pre-installed Python"?